config/log4j2.properties

```diff
@@ -0,0 +1,44 @@
+##############################################################################
```

Comment:
remove this file, since there is the same file in the correct place.

Reply:
done